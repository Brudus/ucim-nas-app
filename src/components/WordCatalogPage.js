import React from 'react';
import WordList from './WordList';
import WordListFilters from './WordListFilters';
import WordsSummary from './WordsSummary';

const WordCatalogPage = () => (
    <div>
        <WordsSummary />
        <WordListFilters />
        <WordList />
    </div>
);

export default WordCatalogPage;