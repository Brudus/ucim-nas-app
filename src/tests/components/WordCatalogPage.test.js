import React from 'react';
import { shallow } from 'enzyme';
import WordCatalogPage from '../../components/WordCatalogPage';

test('should render WordCatalogPage', () => {
    const wrapper = shallow(<WordCatalogPage />);
    expect(wrapper).toMatchSnapshot();
});
