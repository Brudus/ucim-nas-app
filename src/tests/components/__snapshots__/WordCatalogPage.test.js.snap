// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WordCatalogPage 1`] = `
<div>
  <Connect(WordsSummary) />
  <Connect(WordListFilters) />
  <Connect(WordList) />
</div>
`;
